plot/plot.app.js

```diff
@@ -937,8 +963,15 @@
                     // TODO: configure title?
 
                     var studyUri = ERMrest.renderHandlebarsTemplate($rootScope.plot.study_uri_pattern, $rootScope.templateParams);
-
-                    ERMrest.resolve(studyUri, ConfigUtils.getContextHeaderParams()).then(function (ref) {
+                    var headers={}
+                    headers[ERMrest.contextHeaderName]=ConfigUtils.getContextHeaderParams();
+                    var uriParams=studyUri.split("/")
+                    if(uriParams[uriParams.indexOf("entity")+1])
+                        headers[ERMrest.contextHeaderName].schema_table=uriParams[uriParams.indexOf("entity")+1]
+                    if(uriParams[uriParams.indexOf("catalog")+1])
```

Comment:
Same comment I made in the gene_selector case.

Reply:
Made applicable changes